packages/emmett-expressjs/src/e2e/decider/applicationLogicWithOC.int.spec.ts

```diff
@@ -44,7 +44,6 @@ describe('Application logic with optimistic concurrency', () => {
 
     if (!current.id) {
       assert.fail();
-      return;
```

Comment:
when we call `assert.fail()`, node automatically throws an assertion error, so this line was unreachable

Reply:
ah, didn't know that. Thanks for the hint!